### PR TITLE
condense various if conditions in a single condition

### PR DESCRIPTION
**Pull request by boddumanohar**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:
Here I attempt to condense various if-statements into a single statements. Both the changes are trying to do the same. i.e `return false if any of the condition fails`. This can be better expressed in a single statement that I propose rather than using multiple statements. 
 
**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by boddumanohar** [comment]
cc:@justinsb

File: pkg/controller/service/controller.go [hunk 1]
```diff
@@ -564,26 +564,13 @@ func portSlicesEqualForLB(x, y []*v1.ServicePort) bool {
 
 func portEqualForLB(x, y *v1.ServicePort) bool {
 	// TODO: Should we check name?  (In theory, an LB could expose it)
-	if x.Name != y.Name {
+	if x.Name != y.Name &&
```
Comment (andrewsykim) [inline pkg/controller/service/controller.go]: I think these should be `||`?

**Review by andrewsykim — COMMENTED** [review]
For readability I actually like the if conditions separated.

File: pkg/controller/service/controller.go (hunk 1 above)
Comment (boddumanohar) [reply pkg/controller/service/controller.go]: sorry. my bad.

Comment (boddumanohar) [reply pkg/controller/service/controller.go]: updated.

**Comment by boddumanohar** [comment]
@andrewsykim thanks for letting me know your opinion. I will close the PR.

**Pull request closed without merging**